### PR TITLE
extract bookmark functionality from characters state

### PR DESCRIPTION
**Pull request by kai-franke**
Hello everyone!

I have created a second state for the favorites/bookmarks and adjusted the functionality so that I no longer store the characters in the localStorage. If someone has time I would be happy about an approval.

Cheers,
Kai

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -9,61 +9,60 @@ import RandomPage from './pages/RandomPage';
...
-      fetchCharacters();
-    }
+    fetchCharacters();
   }, []);
 
   useEffect(() => {
-    localStorage.setItem('characters', JSON.stringify(characters));
-  }, [characters]);
+    localStorage.setItem('favorites', JSON.stringify(favorites));
+  }, [favorites]);
```
Comment (Roland-Hufnagel) [inline src/App.js]: Why don't you use your useLocalStorage Hook?

**Review by Roland-Hufnagel — APPROVED** [review]
Good solution with just a few components. Great job.

File: src/App.js (hunk 1 above)
Comment (kai-franke) [reply src/App.js]: Thank you for reminding me. I just pushed the revised version. :D

**Review by CathrinTruchan — APPROVED** [review]
Nice! Good Job :)

**Pull request merged**